Back-end/prisma/migrations/20240702201546_comments/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- CreateTable
+CREATE TABLE "Comments" (
+    "comment_id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "Post_id" INTEGER NOT NULL,
```

Comment:
similar here. The table column name should use either camel case or sneak case, but not mixed

Reply:
Whenever I make new changes to the schema, the migration.sql creates that file. I am not sure if I can make changes to it since it is connected to the ORM.